CONTRIBUTING.md

```diff
@@ -136,17 +136,15 @@ When opening a pull request, please use at least one of the following prefixes:
 * **[REF]** for refactoring existing code
 * **[STY]** for stylistic changes
 * **[TST]** for new or updated tests, and
-* **[WIP]** for changes which are not yet ready to be merged
+* **[DPR]** for creating draft pull requests
```

Comment:
Per [this comment](https://github.com/ME-ICA/tedana/pull/244#issuecomment-478255157), please undo this change.